### ISSUE TITLE
Clipboard spoiled

### ISSUE THREAD

**Issue by QuAzI** [symless/synergy#5659]
### Operating Systems ###

Client: Linux Mint 17.3

Server: Windows 7 Pro SP1 AMD64 EN

### Synergy Version ###

1.8.4-stable-a6ff907 on both sides

### Steps to reproduce bug ###

Periodically copy/paste doesn't work from Linux host to Windows host. Clipboard empty or have a trash like on screenshot

![untitled_-_notepad_notepad_2016-10-12_18-05-32_g2imb04w](https://cloud.githubusercontent.com/assets/1897605/19315507/9f5dc0c8-90a6-11e6-8b7e-0f004891cba8.png)

### Other info ###

[synergy.log.txt](https://github.com/symless/synergy/files/524904/synergy.log.txt)

**Comment by nlyan** [comment]
Did you experience this problem with v1.8.3 ?

**Comment by QuAzI** [comment]
Yes. Periodically this happens in all versions which I use. Synergy restart on both sides helps before.

**Comment by XinyuHou** [comment]
The empty clipboard issue may be related to #5662

**Comment by denormalizer** [comment]
Agree with @QuAzI . This has happened to me before in previous versions.

Hasn't happened in a while. But maybe because I haven't triggered it somehow.

**Comment by jporkka** [comment]
I see this happen reliably with Slickedit running on Ubuntu 16.04 LTS.
Slickedit puts several formats of text onto the clipboard including plain text, unicode text and HTML.
The HTML format text is what is getting confused on the Windows side.
Using http://www.freeclipboardviewer.com/ on Windows to see the clipboard and its various formats.

Anyways, when I copy text from Slickedit running on Windows, the HTML format clipboard looks OK.


If I copy the same text from Slickedit on Ubuntu, then in the clipboard on Windows, the HTML shows up looking very confused.

![image](https://cloud.githubusercontent.com/assets/2498088/23384576/7a48d07c-fd00-11e6-8824-851b82fb9543.png)
[ClipboardData.zip](https://github.com/symless/synergy/files/805286/ClipboardData.zip)
